tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/EmailSteps.cs

```diff
@@ -37,46 +44,116 @@ public async Task EmailIsNotSent()
             emails.Should().BeNullOrEmpty();
         }
 
-        [Then(@"the email sent contains the following information")]
-        public async Task ThenEmailContains(Table table)
+        [Then(@"the email contains the following information")]
+        public async Task ThenTheEmailContainsTheFollowingInformation(Table table)
         {
             var expectedContents = table.CreateSet<EmailContentsTable>().First();
 
             var expected = new
             {
-                Attachments = new List<EmailAttachmentData>
-                {
-                    new EmailAttachmentData(
-                        Encoding.UTF8.GetBytes($"{expectedContents.AttachmentHeader1}\r\n{expectedContents.OrderId}\r\n"),
-                        expectedContents.FileName,
-                        new ContentType("text/csv"))
-                },
-                From = new List<EmailAddress>
-                {
-                    new EmailAddress(Empty, expectedContents.From)
-                },
+                From = new List<EmailAddress> { new EmailAddress(Empty, expectedContents.From) },
+                To = new List<EmailAddress> { new EmailAddress(Empty, expectedContents.To) },
                 expectedContents.Text,
                 expectedContents.Subject,
-                To = new List<EmailAddress>
-                {
-                    new EmailAddress(Empty, expectedContents.To)
-                }
             };
-            
+
             var actual = (await _emailServerDriver.FindAllEmailsAsync()).First();
 
             actual.Should().BeEquivalentTo(expected, conf => conf.IncludingAllDeclaredProperties());
         }
 
+        [Then(@"the patient numbers price type attachment contains the correct information")]
+        public async Task ThenThePatientNumbersPriceTypeAttachmentContainsTheCorrectInformation()
+        {
+            string[] patientNumbersCsvHeader = {
+                "Call Off Agreement ID",
+                "Call Off Ordering Party ID",
+                "Call Off Ordering Party Name",
+                "Call Off Commencement Date",
+                "Service Recipient ID",
+                "Service Recipient Name",
+                "Service Recipient Item ID",
+                "Supplier ID",
+                "Supplier Name",
+                "Product ID",
+                "Product Name",
+                "Product Type",
+                "Quantity Ordered",
+                "Unit of Order",
+                "Price",
+                "Funding Type",
+                "M1 planned (Delivery Date)",
+                "Actual M1 date",
+                "Buyer verification date (M2)",
+                "Cease Date"
+            };
+
+            var actual = (await _emailServerDriver.FindAllEmailsAsync()).First().Attachments.First();
+
+            var attachmentData = new MemoryStream(actual.AttachmentData.ToArray());
```

Comment:
Should the attachmentData stream be disposed?

Reply:
ooo yes. Should probably use an using here and that should sort it out I think